CMakeLists.txt

```diff
@@ -114,12 +114,12 @@ configure_file(./omrversionstrings.CMakeTemplate.h omrversionstrings.h)
 ###
 include(OmrDDRSupport)
 
-if(NOT DEFINED OMR_DDR_SET)
-	set(OMR_DDR_SET "omr")
-	make_ddr_set(${OMR_DDR_SET})
+make_ddr_set(omrddr)
+# Note  this is a temporary workarround for consumers which still use teh OMR_DDR_SET mechanic
```

Comment:
typo: teh -> the; mechanic -> mechanism

Reply:
These typos are not fixed; also workarround -> workaround.

Reply:
This is an outdated diff, it is not in the final PR